WaylonWalker/markata
# fix/relative icon should be absolute

- feat:service-worker plugin
- docs
- need to allow precache to be applied to feed
- format with black
- use latest version of hatch-actio
- update docs
- update changelog
- remove broken script
- fix: icon link should be absolute not relative
